feat: standardize view button text

Hdanzo opened the pull request:
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46497

Add new translation keys 
Delete unused translation keys 
Fix tests broken with new functionalities

Commit:
feat: standardize view button text

Add new translation keys Delete unused translation keys Fix tests broken with new functionalities

Sboonny commented:
I have closed this and reopened it, because cypress test looks weird.

Sboonny reviewed (approved):
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Hdanzo, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

Sboonny commented on client/i18n/locales/english/translations.json:
```diff
@@ -6,12 +6,12 @@
     "frontend": "Front End",
     "backend": "Back End",
     "view": "View",
+    "view-code": "View <0>Code</0>",
+    "view-solution": "View <0>Solution</0>",
+    "view-project": "View <0>Project</0>",
```
There is may cause confusion to screen readers, and translation team

```suggestion
```

Sboonny commented on client/src/components/solution-display-widget/index.tsx:
```diff
@@ -27,9 +27,13 @@ export function SolutionDisplayWidget({
   const showOrViewText =
-    displayContext === 'settings'
-      ? t('buttons.show-solution')
-      : t('buttons.view');
+    displayContext === 'settings' ? (
+      <Trans i18nKey='buttons.view-solution'>
+        <span className='sr-only'>Solution</span>
+      </Trans>
+    ) : (
+      t('buttons.view')
+    );
```
with deletion of other view variation, we can have simple translated view text
```suggestion
 const viewText = t('buttons.view')
```

Sboonny commented on client/src/components/solution-display-widget/index.tsx:
```diff
@@ -131,6 +142,7 @@ export function SolutionDisplayWidget({
+      title='project link'
       target='_blank'
     >
       {showOrViewText}
```
```suggestion
      {viewText}

Sboonny commented on client/src/components/solution-display-widget/index.tsx:
```diff
@@ -70,10 +74,13 @@ export function SolutionDisplayWidget({
       onClick={showUserCode}
     >
-      {t('buttons.show-code')}
+      <Trans i18nKey='buttons.view-code'>
+        <span className='sr-only'>Code</span>
+      </Trans>
```
```suggestion
      {viewText}
```

Sboonny commented on client/src/components/solution-display-widget/index.tsx:
```diff
@@ -86,10 +93,14 @@ export function SolutionDisplayWidget({
       >
         <MenuItem bsStyle='primary' onClick={showUserCode}>
-          {t('buttons.show-code')}
+          <Trans i18nKey='buttons.view-code'>
+            <span className='sr-only'>Code</span>
+          </Trans>
```
```suggestion
      {viewText}
```

Sboonny commented on client/src/components/solution-display-widget/index.tsx:
```diff
@@ -86,10 +93,14 @@ export function SolutionDisplayWidget({
         </MenuItem>
         <MenuItem bsStyle='primary' onClick={showProjectPreview}>
-          {t('buttons.show-project')}
+          <Trans i18nKey='buttons.view-project'>
+            <span className='sr-only'>Project</span>
+          </Trans>
```
```suggestion
      {viewText}
```

Sboonny reviewed (changes requested):
I had a meeting with @ahmadabdolsaheb, and we discussed the pull request and there are issues and I didn't notice

Sboonny commented on client/src/components/solution-display-widget/index.tsx:
```diff
@@ -70,10 +74,13 @@ export function SolutionDisplayWidget({
       bsStyle='primary'
       className='btn-invert'
       data-cy={dataCy}
+      title='view code button'
```
Adding a title will add a tooltip, if we are going to keep it around, we need it to be descriptive of what the button for, like `navigate to solution of this certifcation` and it needs to be translatable.

we can cross that bridge another time, so here is my suggestion

```suggestion
```

If you want to add it, you can add descriptive title in `.json` file and use it as a value for the title. `viewText` is a good example

Commit:
Delete View variations in translations.json
Delete buttons tittle
Modify buttons text to be "View" or translated equivalent
Modify tests to query the buttons by role and name

Sboonny reviewed (approved):
LGTM 🎉, thank you for your contribution to the page! 👍.
